### PR TITLE
Update changelog for 1.4.0-insiders

### PR DESCRIPTION
**Pull request by sean-mcmanus**

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
 
+## Version 1.4.0-insiders: May 10, 2021
+### Enhancements
+* Update to clang-format 12. [#6434](https://github.com/microsoft/vscode-cpptools/issues/6434)
+* Fix file:line path for $FILEPOS [#7193](https://github.com/microsoft/vscode-cpptools/issues/7193)
+	* [#1124](https://github.com/microsoft/MIEngine/pull/1124)
+* Add `stopAtConnect` and `hardwareBreakpoints` launch options [PR #7449](https://github.com/microsoft/vscode-cpptools/pull/7449) 
+  * `stopAtConnect` stops the debugger on connection to a remote target [PR MIEngine#1109](https://github.com/microsoft/MIEngine/pull/1109)
+  * `hardwareBreakpoints` controls usage and number of remote hardware breakpoints [PR MIEngine#1128](https://github.com/microsoft/MIEngine/pull/1128)
+* Adds support for loading Concord extensions to the cppvsdbg debug adapter (see [documentation](https://github.com/microsoft/ConcordExtensibilitySamples/wiki/Support-for-VS-Code-cppvsdbg-Scenarios) for more information)
```
Comment (michelleangela) [inline Extension/CHANGELOG.md]: Change "Adds" to "Add". Lines 11 and 12

File: Extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+
+### Bug Fixes
+* Fix IntelliSense configuration with cl.exe compiler args `/external:I`, `/Zc:preprocessor`, and others. [#4980](https://github.com/microsoft/vscode-cpptools/issues/4980), [#6531](https://github.com/microsoft/vscode-cpptools/issues/6531), [#7259](https://github.com/microsoft/vscode-cpptools/issues/7259)
+* Fix -idirafter directories being included too early. [#7129](https://github.com/microsoft/vscode-cpptools/issues/7129)
+* Filter out C++ std when querying the compiler as C (and vice versa). [#7269](https://github.com/microsoft/vscode-cpptools/issues/7269)
+* Fix files.exclude ending with `/folder/**` not excluding `/folder`. [#7331](https://github.com/microsoft/vscode-cpptools/issues/7331)
+* Fix VS Code UI freezing when hovering over very large literals. [#7334](https://github.com/microsoft/vscode-cpptools/issues/7334)
+* Fix clang-format formatting bug when new lines are removed. [#7360](https://github.com/microsoft/vscode-cpptools/issues/7360)
+* Change default cwd in launch.json to "${fileDirname}. [#7362](https://github.com/microsoft/vscode-cpptools/issues/7362)
```
Comment (michelleangela) [inline Extension/CHANGELOG.md]: `"${fileDirname}` is missing `"` at the end. Or is this supposed to use code markup?

File: Extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+  * Syed Ahmad (@HackintoshwithUbuntu) [PR #7363](https://github.com/microsoft/vscode-cpptools/pull/7363)
+* Fix extension activation with 32-bit Windows. [#7368](https://github.com/microsoft/vscode-cpptools/issues/7368)
```
Comment (michelleangela) [inline Extension/CHANGELOG.md]: Wasn't "Fix extension activation with 32-bit Windows." fixed in 1.3.1? It's also listed under the 1.3.1 changes.

**Review by michelleangela — CHANGES_REQUESTED** [review]

File: Extension/CHANGELOG.md (hunk 3 above)
Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: Yeah, good catch.

**Review by michelleangela — APPROVED** [review]

**Pull request merged**